mantidproject/mantid
# Ensure HKL labels within instrument view miniplot remain visble when zooming

**Description of work.**

See commit message details for more detail than the title suggests.

A commit is included that removes the old Qt4-based code.

**To test:**

* See the instructions in the linked issue and ensure the label remains visible as the
   zoom level changes

Fixes #32084

*This does not require release notes* because **fill in an explanation of why**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
